content/whats-new/Serenity-news.json

```diff
@@ -1,4 +1,11 @@
 [
+  {
+    "title":"Tailored exports 🪡",
```

Comment:
What's that symbol before the last parenthesis?

Reply:
It's a needle! For the "tailored" part. :)